source/guide/3.2/reference-diamond-snmp-integration.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: bt_wiki
+title: Diamond SNMP integration
+category: Reference
+publish: true
+abstract: "Reference on how to use diamond monitoring with SNMP device"
+pageord: 1020
+---
+
+{%summary%}
+An extension to the diamond plugin that adds support for monitoring SNMP metrics on remote machines.
+The example usage and necessary types are located in [cloudify-diamond-snmp-extension](https://github.com/cloudify-cosmo/cloudify-diamond-snmp-extension).
+{%endsummary%}
+
+{%note title=Note%}
+See [Diamond plugin](plugin-diamond.html) for general Diamond plugin usage.
+{%endnote%}
+
+# SNMP types
+All node types you will need are defined in [snmp-types.yaml](https://github.com/cloudify-cosmo/cloudify-diamond-snmp-extension/blob/CFY-2305-snmp_diamond_integration/snmp-types.yaml). Our example utilizes OpenStack, please see [openstack-snmp-types.yaml](https://github.com/cloudify-cosmo/cloudify-diamond-snmp-extension/blob/CFY-2305-snmp_diamond_integration/openstack-snmp-types.yaml) for reference. It contains a security group allowing UDP communication on port 161 (default for SNMP). SNMP proxy is a node responsible for gathering the requested metrics from SNMP devices and sending them to RabbitMQ on behalf of those devices as if they were reporting those metrics by themselves (the proxy should be transparent).
+
+## snmp_monitored_host
+snmp_monitored_host exists in the sample blueprints only as a simulation of a monitored device. We assume that the device runs some SNMP agent (snmpd in our examples) and that the SNMP proxy can access it. In our examples the snmp_monitored_host is a virtual machine with Ubuntu installed on it. The snmpd_configuring_node (see blueprints) installs the SNMP daemon (snmpd) and modifies its configuration so that it can be polled for metrics from anywhere.
+
+## snmp_proxy and snmp_manager_proxy
```

Comment:
instead of `snmp_proxy` and `snmp_manager_proxy` use the types names: `SNMPManagerProxy` and `SNMPProxy`